packages/js/README.md

```diff
@@ -50,7 +50,7 @@ client.connect();
 //  client.disconnect();
 ```
 
-> See [TelnyxRTC#constructor](./docs/ts/classes/telnyxrtc.md#constructors) for all options.
+> See [TelnyxRTC#constructor](./docs/ts/classes/telnyxrtc.md#constructor) for all options.
```

Comment:
Not a TS doc issue, but noticed this was linked to the top "Constructors" header